node/actors/executor/src/lib.rs

```diff
@@ -113,17 +124,17 @@ impl Executor {
         let dispatcher = Dispatcher::new(consensus_dispatcher_pipe, network_dispatcher_pipe);
 
         tracing::debug!("Starting actors in separate threads.");
-        scope::run!(ctx, |ctx, s| async {
+        scope::run!(ctx, |ctx, s| async move {
             s.spawn(async { dispatcher.run(ctx).await.context("IO Dispatcher stopped") });
 
             let (net, runner) =
                 network::Network::new(network_config, self.block_store.clone(), network_actor_pipe);
             net.register_metrics();
             s.spawn(async { runner.run(ctx).await.context("Network stopped") });
 
-            if let Some(debug_config) = &self.config.debug_page {
-                s.spawn(async {
-                    http::Server::new(debug_config.clone(), net)
+            if let Some(debug_config) = self.debug_page_config() {
+                s.spawn(async move {
```

Comment:
don't use "async move" - for moving values inside the closures, use "let x = x;" syntax, or ctx::NoCopy wrapper for passing Copyable objects.